### ISSUE TITLE
the key ['loss'] in the loss_dict

### ISSUE THREAD

**Issue by diaodeyi** [eladrich/pixel2style2pixel#127]
In line 108 and 109 of coach ,i think there is some key error in the " val_loss_dict['loss']"

**Comment by yuval-alaluf** [comment]
Hmm. I'm a bit surprised that you got an error there. Any chance you can provide more details? 
I'm assuming you're working with an older version of the code, but are you referring to these lines?
https://github.com/eladrich/pixel2style2pixel/blob/72f0c04a890adb9c8a406f879980572ac600cf29/training/coach.py#L100-L102

**Comment by diaodeyi** [comment]
> Hmm. I'm a bit surprised that you got an error there. Any chance you can provide more details?
> I'm assuming you're working with an older version of the code, but are you referring to these lines?
> https://github.com/eladrich/pixel2style2pixel/blob/72f0c04a890adb9c8a406f879980572ac600cf29/training/coach.py#L100-L102

yes， i referring to these line. And i add the loss key in the dictionary , the probelm figured out
